test/functional/apps/discover/_large_string.js

```diff
@@ -0,0 +1,67 @@
+import expect from 'expect.js';
+
+export default function ({ getService, getPageObjects }) {
+  const esArchiver = getService('esArchiver');
+  const log = getService('log');
+  const retry = getService('retry');
+  const PageObjects = getPageObjects(['common', 'home', 'settings', 'discover']);
+
+  describe('test large strings', function () {
+    before(async function () {
+      await esArchiver.load('empty_kibana');
+      await esArchiver.loadIfNeeded('hamlet');
+      await PageObjects.settings.navigateTo();
+      await PageObjects.settings.clickKibanaIndices();
+      await PageObjects.settings.createIndexPattern('testlargestring', null);
```

Comment:
I'm pretty sure these lines are redundant and should be removed to speed the test up;
`await PageObjects.settings.navigateTo();`
`await PageObjects.settings.clickKibanaIndices();`

because if you look at `PageObjects.settings.createIndexPattern` the first steps are those 2 ^`